README.rst

```diff
@@ -0,0 +1,98 @@
+.. role:: bash(code)
+   :language: bash
+
+=========
+pyUmbral
+=========
+
+.. image:: https://travis-ci.org/nucypher/pyUmbral.svg?branch=master
+    :target: https://travis-ci.org/nucypher/pyUmbral
+
+pyUmbral is a 100% python implementation of Umbral using OpenSSL and Cryptography.io,
+enabling users to perform *split-key proxy-rencryption* and public key encryption
+in an understandable and usable manner.
+
+**Alice encrypts data for herself**
+
+.. code-block:: python
+
+    from umbral import umbral, keys
+
+    # Generate a key pair for Alice
+    alices_private_key = keys.UmbralPrivateKey.gen_key()
+    alices_public_key = private_key.get_pubkey()
+
+    # Encrypt data with bob's public key
+    plaintext = b'Proxy Re-encryption is cool!'
+    ciphertext, capsule = umbral.encrypt(alices_public_key, plaintext)
+
+    # Decrypt data with Bob's keys
+    cleartext = umbral.decrypt(capsule, alices_private_key, ciphertext, alices_public_key)
+
+**Ursula Re-encrypts for Bob**
+
+.. code-block:: python
+
+    # Generate a key pair for Bob
+    bobs_private_key = keys.UmbralPrivateKey.gen_key()
+    bobs_public_key = private_key.get_pubkey()
+
+    # Bob receives the capsule through a side channel (S3, ipfs, Google Cloud, etc.)
+    capsule = capsule
+
+    # Generate split re-encryption keys with "M of N"
+    # Minimum threshold of 10 ("M") - and 20 total shares ("N").
+    kfrags, _ = umbral.split_rekey(alices_private_key, bobs_public_key, 10, 20)
+
+    # Ursula re-encrypts the shares and bob attaches them to the capsule
+    for kfrag in kfrags:
+        cfrag = umbral.reencrypt(kfrag, umbral_capsule)
+        bob_capsule.attach_cfrag(cfrag)
```

Comment:
what`s `umbral_capsule` and `bob_capsule`? There is an issue here with the name of the capsules. I think all should have distinctive names (e.g., alices_capsule, bob_capsule)

Reply:
Hmm, within the context of this story, isn't it the same capsule?

Reply:
oh yes, we have had so many different versions of the capsule concept that I forgot that right now we only have one capsule that gets enriched with cfrags.
OK, then you're right, both should be the same.  Perhaps just `capsule` in both cases?

Reply:
:heavy_check_mark: